docs/notifications/events/components.md

```diff
@@ -0,0 +1,75 @@
+| `configs`            | The configs of the component                         | `map[string]string` | `true`   |
+| `component_checks`   | The component checks of the component                | `map[string]string` | `true`   |
+| `properties`         | The properties of the component                      | `map[string]string` | `true`   |
+| `path`               | The path of the component                            | `string`            |          |
+| `summary`            | The summary of the component                         | `map[string]string` | `true`   |
+| `is_leaf`            | Whether the component is a leaf                      | `bool`              |          |
+| `cost_per_minute`    | The cost per minute of the component                 | `float64`           |          |
+| `cost_total_1d`      | The cost total 1d of the component                   | `float64`           |          |
+| `cost_total_7d`      | The cost total 7d of the component                   | `float64`           |          |
+| `cost_total_30d`     | The cost total 30d of the component                  | `float64`           |          |
+| `created_by`         | Id of the person that created this component         | `uuid`              |          |
+| `created_at`         | Created timestamp                                    | `time.Time`         |          |
+| `updated_at`         | Updated timestamp                                    | `time.Time`         |          |
+| `deleted_at`         | Deleted timestamp                                    | `time.Time`         | `true`   |
+
+### Agent
+
+| Field         | Description                           | Schema              | Optional |
+| ------------- | ------------------------------------- | ------------------- | -------- |
+| `id`          | The id of the agent                   | `uuid`              |          |
+| `name`        | The name of the agent                 | `string`            |          |
+| `hostname`    | The hostname of the agent             | `string`            |          |
+| `description` | Short description of the agent        | `string`            |          |
+| `ip`          | The ip of the agent                   | `string`            |          |
+| `version`     | The version of the agent              | `string`            |          |
+| `username`    | The username of the agent             | `string`            |          |
+| `person_id`   | The person associated with this agent | `uuid`              | `true`   |
+| `properties`  | The properties of the agent           | `map[string]string` | `true`   |
+| `tls`         | The tls of the agent                  | `string`            |          |
+| `created_by`  | The created by of the agent           | `uuid`              |          |
+| `created_at`  | The created at of the agent           | `time.Time`         |          |
+| `updated_at`  | The updated at of the agent           | `time.Time`         |          |
```

Comment:
Do we have all of these fields? It probably also doesn't make sense to haver more than id and name

Reply:
okay. getting rid of them. We can always add them if the need arises.